packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -124,284 +112,302 @@ const initNobleAccount = async (
 };
 
 const initRemoteEVMAccount = async (
-  orch: Orchestrator,
+  orch,
+  seat,
+  ctx: PortfolioInstanceContext,
```

Comment:
re argument order: orchestration flows always start with `(orch, ctx, ...)`. We're not currently using this as a flow, but I can imagine we might want to. Plus, just for the sake of consistency, I **suggest**:

```suggestion
  orch,
  ctx: PortfolioInstanceContext,
  seat,
```